index.js

```diff
@@ -1600,30 +1604,65 @@ Auth0.prototype.requestSMSCode = function (options, callback) {
     throw new Error('A callback function is required');
   }
 
-  assert_required(options, 'apiToken');
   assert_required(options, 'phone');
+  options.phone_number = options.phone;
+  delete options.phone;
+
+  return this.startPasswordless(options, callback);
+};
+
+/**
+ * Send Email to do passwordless authentication
+ *
+ * @example
+ *
+ *     auth0.startPasswordless({email: 'foo@bar.com'}, function (err, result) {
+ *       if (err) return console.log(err.message);
+ *       console.log(result);
+ *     });
+ *
+ * @method startPasswordless
+ * @param {Object} options
+ * @param {Function} callback
+ */
 
-  var apiToken = options.apiToken;
-  var phone = options.phone;
+Auth0.prototype.startPasswordless = function (options, callback) {
+  if ('object' !== typeof options) {
+    throw new Error('An options object is required');
+  }
+  if ('function' !== typeof callback) {
+    throw new Error('A callback function is required');
+  }
+  if (!options.email && !options.phone_number) {
+    throw new Error('An email or a phone number is required.');
+  }
 
   var protocol = 'https:';
   var domain = this._domain;
-  var endpoint = '/api/v2/users';
+  var endpoint = '/passwordless/start';
   var url = joinUrl(protocol, domain, endpoint);
 
+  var data = {client_id: this._clientID};
+  if (options.email) {
+    data.email = options.email;
+    data.connection = 'email';
+    if (options.authParams) {
+      data.authParams = options.authParams;
+    }
+    if (options.send) {
+      data.send = options.send;
+    }
+  } else {
+    data.phone_number = options.phone_number;
+    data.connection = 'sms';
+  }
+
   return reqwest({
     url:          same_origin(protocol, domain) ? endpoint : url,
     method:       'post',
     type:         'json',
     crossOrigin:  !same_origin(protocol, domain),
-    headers:      {
-      Authorization: 'Bearer ' + apiToken
-    },
-    data:         {
-      phone_number:   phone,
-      connection:     'sms',
-      email_verified: false
-    }
+    data:         data
```

Comment:
@gnandretta also, given this enpoint we could now support for `jsonp`, am I right?